src/utils/db.js

```diff
@@ -0,0 +1,6 @@
+import { Sequelize } from 'sequelize';
+
+export const sequelize = new Sequelize('postgres', 'postgres', 'qwerty1234', {
```

Comment:
better to create constants
```suggestion
const DATABASE = 'postgres';
const USERNAME = 'postgres';
const PASSWORD = 'qwerty1234';

export const sequelize = new Sequelize(DATABASE, USERNAME, PASSWORD, {
```